cl_sii/libs/xml_utils.py

```diff
@@ -306,7 +307,7 @@ def validate_xml_doc(xml_schema: XmlSchema, xml_doc: XmlElement) -> None:
         # Simplest and safest way to get the error message.
         # Error example:
         #   "Element 'DTE': No matching global declaration available for the validation root., line 2"  # noqa: E501
-        validation_error_msg = str(exc)
+        validation_error_msg = str(exc) + f". cl-sii Python lib version {__version__}"
```

Comment:
Perhaps add an optional parameter to the function for specifying the `xml_schema` version (string) (however the caller chooses to define it)